.travis.yml

```diff
@@ -136,3 +135,12 @@ install: true  # Skip the installation step, as Maven requires
 script:
   - cd "$TRAVIS_BUILD_DIR"
   - .travis/run_travis_job.sh
+
+notifications:
+  email:
+    - if: type=cron
+    - on_success: always
```

Comment:
Will the default (on_failure I suppose) email notification remain for branch builds?